### PR TITLE
Add Script Command: search-github

### PR DESCRIPTION
**Pull request by crstauf**
## Description

Search GitHub.

## Type of change

- [x] New script command

## Screenshot

Screen Shot 2020-11-23 at 2 08 51 PM

## Dependencies / Requirements

- `open`

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/web-searches/search-github.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# @raycast.title Search cdnjs
+# @raycast.mode silent
+# @raycast.author Caleb Stauffer
+# @raycast.authorURL https://github.com/crstauf
+# @raycast.description Search cdnjs.com for library.
+# @raycast.icon images/cloudflare-logo.png
+# @raycast.argument1 { "type": "text", "placeholder": "Library" }
+# @raycast.packageName Web Searches
+
+open "https://cdnjs.com/libraries?q=${1// /%20}"
```
Comment (unnamedd) [inline commands/web-searches/search-github.sh:13]: @crstauf I guess you got some confusion here...

Comment (crstauf) [reply commands/web-searches/search-github.sh:13]: Agh, I've been trying Nova lately, and it doesn't open the new file when duplicating an existing file. I've done this sort of thing at least a dozen times.

**Review by fe9lix — CHANGES_REQUESTED** [review]
Could you update the logo? It's hardly visible in light mode (you can change the appearance in Raycast preferences for testing).

**Comment by crstauf** [comment]
@fe9lix I was directed by @thomaspaulmann (https://github.com/raycast/script-commands/pull/31#issuecomment-723248908) to use icons for dark theme until support for both themes is available.

**Comment by fe9lix** [comment]
@crstauf Alright, then let's keep it and add another version when we ship theming support.

**Review by fe9lix — APPROVED** [review]

**Pull request merged**